src/sentry/integrations/slack/integration.py

```diff
@@ -103,32 +124,28 @@ def get_team_info(self, access_token):
         resp.raise_for_status()
         status_code = resp.status_code
         resp = resp.json()
+        # TODO: track_response_code won't hit if we have an error status code
         track_response_code(status_code, resp.get("ok"))
```

Comment:
I can update this to use the client in this PR https://github.com/getsentry/sentry/pull/18346